Cargo.toml

```diff
@@ -45,58 +49,61 @@ payjoin = { version = "0.6.0", features = ["sender"] }
 pretty_env_logger = "0.4.0"
 regex = "1.7.0"
 reqwest = { version = "0.11.13", features = ["json"] }
+rgb-std = { version = "0.10.0-rc.1" }
+rgb-wallet = {version = "0.10.0-beta.1" }
```

Comment:
Looks like a few of these deps need to be updated already...
rgb-std is in rc.2 now, and rgb-wallet is in rc.1.

Reply:
Yes, Maxim launch final release today!

I will update ASAP.